### ISSUE TITLE
There is no help for 'ctl' but errors in use tell you to look at the help.

### ISSUE THREAD

**Issue by shaleh** [brson/multirust#86]
I opened a ticket instead of a pull request because I was not sure what the right thing to do here. DO you intend ctl to be a psuedo hidden feature?

**Comment by brson** [comment]
Yes, indeed. It's intended to be a private interface for multirustproxy.

**Comment by shaleh** [comment]
Might want to remove the mention of help then. "This is private, in here be dragons".

**Comment by brson** [comment]
Yes, good idea.
